### ISSUE TITLE
have you seen ktorn/docker-scuttlebot?

### ISSUE THREAD

**Issue by dominictarr** [asbjornenge/patchwork-docker#1]
did you also see https://github.com/ktorn/docker-scuttlebot it looks like these might be pretty similar.

(also, if you are not already on, would you like a invite to join the network?)

**Comment by dominictarr** [comment]
oh, I should mention @ktorn so you can discuss

**Comment by asbjornenge** [comment]
@dominictarr oh, nice! I didn't see that. I searched the docker hub for `patchwork` but I see now it's `scuttlebot` I should have searched for :stuck_out_tongue_winking_eye: 

Yeah, the two `Dockerfiles` produce pretty much the same container. I'm happy to point people to @ktorn's version, but I'll be happy to keep mine up (& up-to-date) also, in case others make the same mistake I did :wink: 

I think I'm on the network already! I did get an invite on #scuttlebut on freenode earlier which populated my Activity Feed considerably :smile:  

I'll just close this issue, but feel free to keep discussing here @ktorn if you have any followup :+1:
